iothub_client/tests/longhaul_resource_cleanup/longhaul_resource_cleanup.c

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "azure_c_shared_utility/xlogging.h"
+#include "azure_c_shared_utility/platform.h"
+#include "azure_c_shared_utility/threadapi.h"
+#include "azure_c_shared_utility/crt_abstractions.h"
+#include "azure_c_shared_utility/shared_util_options.h"
+#include "iothub_device_client.h"
+#include "iothub_client_options.h"
+#include "iothub_message.h"
+#include "iothub_registrymanager.h"
+#include "iothubtransportamqp.h"
+#include "iothub_account.h"
+#include "iothubtest.h"
+#include "parson.h"
+#include "../common_longhaul/iothub_client_common_longhaul.h"
+
+#ifdef SET_TRUSTED_CERT_IN_SAMPLES
+#include "certs.h"
+#endif // SET_TRUSTED_CERT_IN_SAMPLES
+
+const int five_days = (60 * 60 * 24) * 5;
```

Comment:
I have no good comments, you got all covered. So here it goes... please use all upper case for constants.

```suggestion
const int FIVE_DAYS = (60 * 60 * 24) * 5;
```